src/main/play-doc/operation/ClusterUpgrade.md

```diff
@@ -0,0 +1,116 @@
+# Upgrading ConductR Cluster
+
+A running cluster can be upgraded without downtime either at the node or cluster level.
+
+On a per node basis, new updated nodes are introduced to the cluster while the old nodes are removed from the cluster. The per node update strategy applies to both ConductR Core node or ConductR Agent node.
+
+On a per-cluster basis, incoming requests are directed from the old cluster to the new cluster using load balancer membership or DNS changes.
```

Comment:
Which text are you suggesting instead? Note that this page has not been changed by the PR, it has been just moved to a separate page.